CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+### Breaking changes
```

Comment:
We don't consider bumping MSRV a breaking change anymore (see readme). So It shouldn't be in that section in the changelog.

Reply:
Let's put it in "Changed"

Reply:
@JelteF yeah, I do remember, but haven't come up with better naming.

Reply:
Renamed as "Changed".